test/AWS.Deploy.Orchestration.UnitTests/AWSResourceQueryerTests.cs

```diff
@@ -175,6 +175,85 @@ public void SortElasticBeanstalkWindowsPlatforms()
+                {
+                    PlatformArn = "arn:aws:elasticbeanstalk:us-west-2::platform/.NET 6 running on 64bit Amazon Linux 2023/3.1.2",
+                    PlatformBranchName = ".NET 6 running on 64bit Amazon Linux 2023",
+                    PlatformVersion = "3.1.2",
+                    PlatformOwner = "2"
+                },
+                new PlatformSummary
+                {
+                    PlatformArn = "arn:aws:elasticbeanstalk:us-west-2::platform/.NET 6 running on 64bit Amazon Linux 2023/3.0.6",
+                    PlatformBranchName = ".NET 6 running on 64bit Amazon Linux 2023",
+                    PlatformVersion = "3.0.6",
+                    PlatformOwner = "3"
+                },
+                new PlatformSummary
+                {
+                    PlatformArn = "arn:aws:elasticbeanstalk:us-west-2::platform/.NET 6 running on 64bit Amazon Linux 2023/3.0.5",
+                    PlatformBranchName = ".NET 6 running on 64bit Amazon Linux 2023",
+                    PlatformVersion = "3.0.5",
+                    PlatformOwner = "4"
+                },
+                new PlatformSummary
+                {
+                    PlatformArn = "arn:aws:elasticbeanstalk:us-west-2::platform/.NET 8 running on 64bit Amazon Linux 2023/3.1.3",
+                    PlatformBranchName = ".NET 8 running on 64bit Amazon Linux 2023",
+                    PlatformVersion = "3.1.3",
+                    PlatformOwner = "0"
+                },
+                new PlatformSummary
+                {
+                    PlatformArn = "arn:aws:elasticbeanstalk:us-west-2::platform/.NET Core running on 64bit Amazon Linux 2/2.8.0",
+                    PlatformBranchName = ".NET Core running on 64bit Amazon Linux 2",
+                    PlatformVersion = "2.8.0",
+                    PlatformOwner = "5"
+                },
+                new PlatformSummary
+                {
+                    PlatformArn = "arn:aws:elasticbeanstalk:us-west-2::platform/.NET Core running on 64bit Amazon Linux 2/2.7.3",
+                    PlatformBranchName = ".NET Core running on 64bit Amazon Linux 2",
+                    PlatformVersion = "2.7.3",
+                    PlatformOwner = "6"
+                },
+                new PlatformSummary
+                {
+                    PlatformArn = "arn:aws:elasticbeanstalk:us-west-2::platform/.NET Core running on 64bit Amazon Linux 2/2.6.0",
+                    PlatformBranchName = ".NET Core running on 64bit Amazon Linux 2",
+                    PlatformVersion = "2.6.0",
+                    PlatformOwner = "7"
+                },
+                new PlatformSummary
+                {
+                    PlatformArn = "arn:aws:elasticbeanstalk:us-west-2::platform/.NET Core running on 64bit Amazon Linux 2/2.5.7",
+                    PlatformBranchName = ".NET Core running on 64bit Amazon Linux 2",
+                    PlatformVersion = "2.5.7",
```

Comment:
If you change this `PlatformVersion` to 1000.1.0 -> This platform summary shows up on the top of sorted list. This is not intended since we want the all .NET Core platforms to show up after .NET X.

Refer to https://github.com/aws/aws-dotnet-deploy/pull/847/files#r1688563597